feature-libs/product-configurator/common/components/configure-product/configure-product.component.ts

```diff
@@ -31,7 +31,24 @@ export class ConfigureProductComponent {
   ownerTypeProduct: CommonConfigurator.OwnerType =
     CommonConfigurator.OwnerType.PRODUCT;
 
+  preventScrollingOnSpace(event: KeyboardEvent): void {
+    if (event.code === 'Space') {
+      event.preventDefault();
+    }
+  }
+  navigateToConfigurator(event: KeyboardEvent, product: Product): void {
```

Comment:
Please, write a comment for this method.

Reply:
Done